pkg/cmd/pipelineresource/delete.go

```diff
@@ -0,0 +1,66 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipelineresource
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+func deleteCommand(p cli.Params) *cobra.Command {
+	f := cliopts.NewPrintFlags("delete")
+	eg := `
+# Delete a PipelineResource of name 'foo' in namespace 'bar'
+tkn resource delete foo -n bar
+
+tkn res del foo -n bar",
+`
+
+	c := &cobra.Command{
+		Use:          "delete",
+		Aliases:      []string{"del"},
```

Comment:
how about `rm` as well to match `ls` in `list`

Reply:
Thank you very much!!
I didn't realize the list aliases are like that, I'll match it.